modules/op-components.adoc

```diff
@@ -0,0 +1,122 @@
+
+Optionally, you can define an `Interceptor` CR. An interceptor defines the logic that runs every time an event is received through the `EventListener` object, before this event is processed using a trigger. The interceptor can verify, filter, and modify events.
+
+[id="op-overview-catalog_{context}"]
+== Pipelines Catalog, {artifact-hub}, and {tekton-hub}
+
+A _Pipelines catalog_ is a repository of pipelines and tasks. You can download these pipelines and tasks and use them in your CI/CD processes.
+
+You can use the community-sourced Pipelines catalog indexed by the {artifact-hub} and {tekton-hub} projects.
+
+[id="op-overview-resolvers_{context}"]
+== Resolvers
+
+With _resolvers_, you can retrieve tasks and pipelines from a repository by using an identifier. You can reference a task or pipeline by using a resolver instead of defining the task or pipeline in your CRs. When the task or pipeline must be run, {pipelines-shortname} retrieves and runs the task or pipeline.
+
+{pipelines-shortname} includes the following resolvers:
+
+* _Hub resolver_: Retrieves a task or pipeline from the Pipelines Catalog available on the Artifact Hub or Tekton Hub.
+
+* _Bundles resolver_: Retrieves a task or pipeline from a Tekton bundle, which is an Open Container Initiative (OCI) image available from any OCI repository including an OpenShift image registry.
+
+* _Cluster resolver_: Retrieves a task or pipeline that is already created on the same OpenShift cluster in a specific namespace.
+
+* _Git resolver_: Retrieves a task or pipeline definition from a Git repository. You must specify the repository, the branch, and the path.
+
+[id="op-overview-clustertasks_{context}"]
+== Cluster tasks and cluster trigger bindings
+
+When you install {pipelines-shortname}, the installation creates a certain number of `ClusterTask` and `ClusterTriggerBinding` CRs. These resources represent pre-arranged tasks and trigger bindings that you can use in your pipelines.
+
+Cluster tasks and cluster trigger bindings are deprecated and will be removed in a future release. Instead, you can configure the hub resolver to access a wide array of reusable components.
+
+[id="op-overview-chains_{context}"]
+== {tekton-chains}: signed storage of runs for security audit
+
+Any automatic process involving data from repositories is a potential security issue, because a repository might be compromised.
+
+To monitor and mitigate any possible security issues, {pipelines-shortname} provides {tekton-chains}, a method of storing the full contents of every task run and pipeline run, including any data (such as mounted volumes) that was processed in the run. The stored information is signed by a key, so that any tampering with the information is evident.
```

Comment:
```suggestion
To monitor and mitigate any possible security issues, {pipelines-shortname} provides {tekton-chains}, a method of storing the entire content of every task run and pipeline run, including any data (such as mounted volumes) processed in the run. The stored information is signed by a key so that any tampering with the information is evident.
```